app/middlewares/validators/users.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('joi');
```

Comment:
We usually use express-validator for this or even express-openapi-validator. Is there any advantage in using this?

Reply:
I don't know, the reality is that i have experience using joi-validation library, but I gonna change by the express-validator

Reply:
There is no need to do that, it works anyway  :smile: